internal/common/metrics/service_wrapper.go

```diff
@@ -1,392 +0,0 @@
-	scopeNamePollForActivityTask                = TemporalMetricsPrefix + "PollForActivityTask"
-	scopeNamePollForDecisionTask                = TemporalMetricsPrefix + "PollForDecisionTask"
-	scopeNameRecordActivityTaskHeartbeat        = TemporalMetricsPrefix + "RecordActivityTaskHeartbeat"
-	scopeNameRecordActivityTaskHeartbeatByID    = TemporalMetricsPrefix + "RecordActivityTaskHeartbeatById"
-	scopeNameRegisterNamespace                  = TemporalMetricsPrefix + "RegisterNamespace"
-	scopeNameRequestCancelWorkflowExecution     = TemporalMetricsPrefix + "RequestCancelWorkflowExecution"
-	scopeNameRespondActivityTaskCanceled        = TemporalMetricsPrefix + "RespondActivityTaskCanceled"
-	scopeNameRespondActivityTaskCompleted       = TemporalMetricsPrefix + "RespondActivityTaskCompleted"
-	scopeNameRespondActivityTaskFailed          = TemporalMetricsPrefix + "RespondActivityTaskFailed"
-	scopeNameRespondActivityTaskCanceledByID    = TemporalMetricsPrefix + "RespondActivityTaskCanceledById"
-	scopeNameRespondActivityTaskCompletedByID   = TemporalMetricsPrefix + "RespondActivityTaskCompletedById"
-	scopeNameRespondActivityTaskFailedByID      = TemporalMetricsPrefix + "RespondActivityTaskFailedById"
-	scopeNameRespondDecisionTaskCompleted       = TemporalMetricsPrefix + "RespondDecisionTaskCompleted"
-	scopeNameRespondDecisionTaskFailed          = TemporalMetricsPrefix + "RespondDecisionTaskFailed"
-	scopeNameSignalWorkflowExecution            = TemporalMetricsPrefix + "SignalWorkflowExecution"
-	scopeNameSignalWithStartWorkflowExecution   = TemporalMetricsPrefix + "SignalWithStartWorkflowExecution"
-	scopeNameStartWorkflowExecution             = TemporalMetricsPrefix + "StartWorkflowExecution"
-	scopeNameTerminateWorkflowExecution         = TemporalMetricsPrefix + "TerminateWorkflowExecution"
-	scopeNameResetWorkflowExecution             = TemporalMetricsPrefix + "ResetWorkflowExecution"
-	scopeNameUpdateNamespace                    = TemporalMetricsPrefix + "UpdateNamespace"
-	scopeNameQueryWorkflow                      = TemporalMetricsPrefix + "QueryWorkflow"
-	scopeNameDescribeTaskList                   = TemporalMetricsPrefix + "DescribeTaskList"
-	scopeNameRespondQueryTaskCompleted          = TemporalMetricsPrefix + "RespondQueryTaskCompleted"
-	scopeNameDescribeWorkflowExecution          = TemporalMetricsPrefix + "DescribeWorkflowExecution"
-	scopeNameResetStickyTaskList                = TemporalMetricsPrefix + "ResetStickyTaskList"
-	scopeNameGetSearchAttributes                = TemporalMetricsPrefix + "GetSearchAttributes"
-	scopeNameListTaskListPartitions             = TemporalMetricsPrefix + "ListTaskListPartitions"
-	scopeNameGetClusterInfo                     = TemporalMetricsPrefix + "GetClusterInfo"
-	scopeNameGetWorkflowExecutionRawHistory     = TemporalMetricsPrefix + "GetWorkflowExecutionRawHistory"
-)
-
-var (
-	_ workflowservice.WorkflowServiceClient = (*workflowServiceMetricsWrapper)(nil)
```

Comment:
This line was a pain. It is the only place where SDK had a direct dependency on `workflowservice.WorkflowServiceClient` interface.